resources/js/Shared/Comments.vue

```diff
@@ -0,0 +1,202 @@
+<style lang="scss" scoped>
+.box-bottom {
+  border-bottom-left-radius: 11px;
+  border-bottom-right-radius: 11px;
+}
+</style>
+
+<template>
+  <div>
+    <!-- existing comments -->
+    <div v-if="localComments.length > 0">
+      <div v-for="comment in localComments" :key="comment.id" class="flex">
+        <!-- avatar -->
+        <div v-if="comment.author.id">
+          <avatar :avatar="comment.author.avatar" :size="32" :class="'br-100 mr3'" />
+        </div>
+
+        <!-- avatar if the employee is no longer in the system -->
+        <div v-else>
+          <img loading="lazy" src="/img/streamline-icon-avatar-neutral@100x100.png" alt="anonymous avatar" class="br-100 mr3" height="32"
+               width="32"
+          />
+        </div>
+
+        <!-- comment box -->
+        <div v-if="idToEdit != comment.id" class="box bg-white mb4 w-100">
+          <!-- comment -->
+          <div class="bb bb-gray ph3" v-html="comment.content"></div>
+
+          <!-- comment info -->
+          <div class="bg-gray ph3 pv2 f7 box-bottom">
+            <ul class="ma0 list pl0">
+              <li v-if="comment.author.id" class="di" v-html="$t('app.message_comment_written_by', { url: comment.author.url, name: comment.author.name, date: comment.written_at })"></li>
```

Comment:
Unfortunately, this can lead to a XSRF attack, if someone change its name to "<javascript>alert()</javascript>" or something like that.

Reply:
@asbiin how do we do that then? If we need to build a string that will contain a link, we need to have html in the i18n file. But if we do, we need `<v-html` so it's parsed correctly.

Reply:
I know, I don't say it's easy, but in this case, it's a problem.

Reply:
So basically everywhere we do that, it's a problem. Is there a way to sanitize that?